ui/raidboss/popup-text.ts

```diff
@@ -69,14 +64,16 @@ export type ProcessedTrigger = LooseTrigger & {
   output?: Output;
 };
 
-type ProcessedTimelineTrigger = ProcessedTrigger & {
-  isTimelineTrigger?: true;
-};
+// a loaded and localized trigger
+export type LocalizedTrigger =
+  & LooseTrigger
+  & { id: string; filename: string; output?: Output }
+  & ({ localRegex: RegExp } | { localNetRegex: RegExp });
 
-type ProcessedTriggerSet = LooseTriggerSet & {
-  filename?: string;
-  timelineTriggers?: ProcessedTimelineTrigger[];
-  triggers?: ProcessedTrigger[];
+export type LoadedTriggerSet = LooseTriggerSet & {
+  filename: string;
+  timelineTriggers?: Omit<LooseTrigger, 'netRegex'>[];
+  triggers?: LooseTrigger[];
```

Comment:
I think this `timelineTriggers` type is incorrect. Shouldn't it be `LooseTimelineTrigger[]`? And if that's true, aren't these both already the same types in `LooseTriggerSet`?

```suggestion
```